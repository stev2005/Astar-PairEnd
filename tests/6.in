TCAGAGCTCTGGAGAATGTCGTCTCTCAAGCTTCCTTTTCGCTGAAGTGCAAACTCAACAGGTCACCTAAAGCAACATAGAAGCCGGCGACCGCGGGTGGTGACGAGGTTAATTCCCTGTTAAGCTATGGCGCCTTCCGCAGCACGCGCTGCTGTACTTAGAGTCCGTGAATTCTATATATCCTTATCATCGTTCCATAGAACTCATAAGAATGTTGGAGCCACACGACGGCGAATATTCGTC
15
CAAAGAGACTCCATCCCAGTAGGAGCCAAGCGGGGCGACATGCACTCGTACTGTTAACAATGGGAATGCAGAGAATTGCGGGCACACCCCCCTATTAGGCGAGAAGTTAACGGGTTTAAACCCCCAAGGGTATCATATTATATCTGAGGAGTGAATACCTTCTGGAGGTCTTCGAAACGGCCCTCGTCTAAGC
CAATGTACGCTTGTAACCCTCGCAAAAAGGATGAGAAGGGACCGCCGGTACAGGAGGATCGTAAATCACTGCAATCCGAAGCAAAATGCGTTCA
CTCCAAAGAACGCTGGCTGGTTCGATCTACTGAATCATTCTATCACTCACTAAAGATTAACTGCACGACCCAGACGCATGGTGGTATTTTTAGCGGGATTGAAACCGGGCAAAGGAGCTGAGGTTACCCACTCCTGGCAAGCATTTTGAGACAGATTCAAGCTTGGCCTCCCGGAGCTACC
GCGGACACCGGACCGGGCTTTGCTAGTAAAAGGGCGACTCTGGATAGCGCACTGATATTAATTGGAAGCATAGCACGGGATGCTACGCCGCCCATTATCCTCCCGCGATAATTCGTTAAAGGCAGATTTCATGGAATTCTTCGGTAGGAGGTATTTTGGGTGCCAGGACTTAACTTCCCCTTAAGTTCGCACTCTATCAAA
CAAAACCCGACTATGTCAACGAGCAGGAGGTTGTTGAAAGCCCCAACCACGGCCTCTCCCTACGTAATCAGGCTTGTATAGGGTTGCGTGTTTACGCCGGGAATTTCTACTGTCCGTGATCTGAATGCAATGATGAGTTGAGATTGCTACGCCACCTAGTACGCAACGTACAGCTCTCAAGGCTGTACTGCTTTATT
GGCTGTAGCATCACACCGACATACTTATTTCCCGAAGAGTACAAGAGTTGCTGCACAAA
TCAAAAAGCGCCTCGGGACCGGAATACAAGTTTATTAACCGGTGACAGCCTTATATTCTTTTGT
CGTCACAGAGGCGAGTTCTGCTCTGAGCGCAAAACACCGCTCGCCACCCATTTCGGCATAGAAGACGCTAGGCAATACACCTACAGTGTGGCGGTGTCTGCCCTCCGCGGTTCTAGGTACGGGAACTTGGGTATGCAATGGGCACGGTCGATAGAATTTCCC
CATTAAGGGAAGCGACTGCTGATGTGGCTTGCACAACGTAGTGTCTAACGTAGGGCAAGAAACAGCATTTTCGGATCCAGTTGCGCGGCCGCCTCTAGTTCGAAACTCGTACTGG
TATCCGGTCAGGGCCTAGTATTGCGGGCAAGAAGCGATCCTTTCCCCCTACGTTTGCCTGCGGCATTAGACGACTCGAGCCTTATGGATGGAAACATCAGCCACGTGAAACCAACTGATCGGCGGTGG
GACTATGTCGTGAAGAGCTGAAAGGGCCATCGTGCTCTGTGGCG
TGAACGAGGTAAACATGGGAACGTTCCCGTACGACTTCCCACGAGGTACCCCGTAGAGTGCTTTAATTCAAGGGGACGCGTGTGCTACCTTTTGGTGCGTGGCACAAGGCACTTTAGTCATATGGCCATTACCCCGCCATTACGTATGAAGTCCCTGGACCAG
CTCTCCGAGTGCACTAGCTTGAGCGGGTGCCAAGTCACCGCATCCTCTAAGTACAGTGCGTGGACCCCGTTTTGAACCAGCGCCTCTTTACTTTTAAACGTAGGGGTAATGCCTTCAGATTCGGACGAGTGCTAACTTCTGTATGTAGGAGCCATCGCCAAACTACATAGGGCTGA
GGATACACAGTCTTCTAAAGCTGTGCACTATCTAATCCAGTTTGGCGTCGCGCTCTAGATGACCACCCGCT
CTATAGGCATACTCCTTCTGCATCCATAGGCTCCGCTCTTATCTAGTATGGCGCGTGCTAACTCGCTGGGCTCGGCACCATTCCCTCTCATCC
