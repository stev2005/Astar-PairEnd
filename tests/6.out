97
44
84
98
95
26
30
74
53
62
19
78
80
32
41
